src/actions/mainActions.js

```diff
@@ -0,0 +1,40 @@
+import * as types from './actionTypes';
+
+export function setTitleActive() {
+    return {
+        type: types.SET_TITLE_ACTIVE
+    }
+}
+export function setGenreActive() {
+    return {
+        type: types.SET_GENRE_ACTIVE
+    }
+}
+export function setSearchValue(val) {
+    return {
+        type: types.SET_SEARCH_STR,
+        value: val
+    }
+}
+export function sortByReleaseDate() {
+    return {
+        type: types.SORT_RELEASE_DATE
+    }
+}
+export function sortByRating() {
+    return {
+        type: types.SORT_RATING
+    }
+}
+export function fetchMoviesSuccess(movies) {
+    return {
+        type: types.FETCH_MOVIES_SUCCESS,

```

Comment:
A good practice when you have actions is follow the [Flux Standard Action](https://github.com/redux-utilities/flux-standard-action), they define some standards to the body of an action, as example, you have type and for movies you must use `payload` for this case.

Reply:
Okay.